deploy/helm/azure-industrial-iot/values.yaml

```diff
@@ -72,9 +70,6 @@ azure:
 
   # Details of Azure CosmosDB that microservices will use as database.
   # The following CosmosDB containers will be created (<db_name>.<container_name>):
```

Comment:
># The follo [](http://example.com/codeflow?start=2&length=11)

does it mean no cosmosdb containers left and it is now obsolete?

Reply:
No, only job database is remaining, but I think this is not the place to document it, case in point this is why this was not updated/captured.